src/db/schema/hackathon.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  date,
+  integer,
+  pgTable,
+  varchar,
+  boolean,
+  smallint,
+} from "drizzle-orm/pg-core";
+import { user } from "./user";
+import { timestamps } from "./schema.helper";
+import { relations } from "drizzle-orm";
+import { hackathonParticipant } from "./hackathonParticipant";
+
+export const hackathon = pgTable("Hackathons", {
+  id: integer().primaryKey().generatedAlwaysAsIdentity(),
```

Comment:
We should use serial instead of integer here.

Reply:
Agreed...

Reply:
addressed